torch/csrc/jit/codegen/cuda/glfdc/sexpr.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+// Basic data structures for expressions
+
+#include <cstddef>
+#include <cstdint>
+#include "c10/util/variant.h"
+
+namespace glfdc {
+
+using scalar_type = std::int64_t;
+
+// Symbolic operand of subexpression
+struct UnboundValue {
+  std::size_t index_;
```

Comment:
What does this `index_` mean?

Reply:
Added comment, please let me know is it clear enough now.